opencti-platform/opencti-front/src/components/CreateEntityControlledDial.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button } from '@mui/material';
+import React, { FunctionComponent } from 'react';
+import { DrawerControlledDialProps } from '../private/components/common/drawer/Drawer';
+import { useFormatter } from './i18n';
+
+interface CreateEntityControlledDialProps extends DrawerControlledDialProps {
+  entityType: string;
+  color?: 'primary' | 'inherit' | 'secondary' | 'success' | 'error' | 'info' | 'warning';
+  size?: 'small' | 'medium' | 'large';
+  variant?: 'text' | 'contained' | 'outlined';
+  style?: React.CSSProperties;
+}
+
+const CreateEntityControlledDial: FunctionComponent<CreateEntityControlledDialProps> = ({
+  onOpen,
+  entityType,
+  color = 'primary',
+  size = 'small',
+  variant = 'contained',
+  style,
+}) => {
+  const { t_i18n } = useFormatter();
+  const buttonValue = `${t_i18n('Create')} ${t_i18n(entityType)}`;
```

Comment:
I suggest we get rid of the `entity_` prefix in all calls to `<CreateEntityControlledDial />`.
This way the prop `entityType` is _really_ an entity type, and the prefix tweak stays for i18n only

```suggestion
  const buttonValue = `${t_i18n('Create')} ${t_i18n(`entity_${entityType}`)}`;
```

Reply:
translating "Create X" as 2 separate calls to `t_i18n` would force the translation order.
In German for instance, the 2 words would actually be inversed.

One way to solve this is to use variables in the intl token.

```suggestion
  const buttonValue = t_i18n('', {
    id: 'Create ...',
    values: { entity_type: t_i18n(`entity_${entityType}`) },
  });
```

and in the translation files

```
"Create ...": "Create {entity_type}",
```